packages/dnb-eufemia/src/components/date-picker/DatePickerPortal.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {
+  useCallback,
+  useEffect,
+  useLayoutEffect,
+  useState,
+} from 'react'
+import ReactDOM from 'react-dom'
+import { DatePickerProps } from './DatePicker'
+
+type DatePickerPortalProps = React.HTMLProps<HTMLDivElement> & {
+  skipPortal?: DatePickerProps['skipPortal']
+  alignment?: DatePickerProps['alignPicker']
+  targetElementRef?: React.RefObject<HTMLElement>
+}
+
+export default function DatePickerPortal({
+  className,
+  skipPortal,
+  alignment,
+  targetElementRef,
+  children,
+}: DatePickerPortalProps) {
+  const [position, setPosition] = useState({})
+
+  const positionTimeout = React.useRef(null)
+
+  useEffect(() => {
```

Comment:
```suggestion
  useLayoutEffect(() => {
```

I think we can set the position earlier, right after the DOM insertion.
Here we need the `useLayoutEffect` import for SSR compatibility.